filterLists.py

```diff
@@ -9,16 +10,18 @@ def __init__(self):
             does not do anything if there is no files that exist
         """
         print("***Filtering bug reports with white/black lists***\n")
-        delimiter = '\n\n'      # individual bugs in each file would need to be separated by two new lines [YOU CAN CHANGE DELIMITER IF YOU WNAT]
         if path.exists("blacklist.txt"):
-            self.black_data = pd.read_csv('blacklist.txt', sep=delimiter,
-                                          header=None, engine='python')
+            blackfile = open("blacklist.txt", 'r')
+            self.black_data = blackfile.read().split(DELIMITER)
+            blackfile.close()
```

Comment:
use the [`with`](https://stackoverflow.com/questions/1369526/what-is-the-python-keyword-with-used-for) keyword to automatically release system resources (and repeat convention below).